README.md

```diff
@@ -82,25 +116,26 @@ const db = admin.firestore();
 integrify({ config: { functions, db } });
 
 // Automatically delete stale references
-module.exports.deleteReferencesToMaster = integrify({
+module.exports.onCustomerDeleted = integrify({
   rule: 'DELETE_REFERENCES',
   source: {
-    collection: 'master', // <-- This will append {masterId}
+    collection: 'customers', // <-- This will append {masterId}
     // OR
-    collection: 'master/{masterId}', // <-- Can be any string as in Firebase
+    collection: 'customers/{cusomterId}', // <-- Can be any string as in Firebase
   },
   targets: [
     {
-      collection: 'detail1',
-      foreignKey: 'masterId', // Optional: Delete document with matching foreign key
+      collection: 'orders',
+      foreignKey: 'customerId', // Optional: Delete document with matching foreign key
       deleteAll: false, // Optional: Delete all from collection
       // EITHER 'foreignKey' OR 'deleteAll' MUST BE PROVIDED
       isCollectionGroup: true,  // Optional: Delete from collection group, see more below
     },
     {
-      collection: 'detail1/$master/details', // Can reference source ID, will throw error if it doesn't exist
+      collection: 'orders/$master/delivered', // Can reference source ID, will throw error if it doesn't exist
       // OR
-      collection: 'detail1/$source.fieldValue/details', // Can reference a field value (requires source), will throw error if it doesn't exist
+      collection: 'orders/$source.fieldValue/delivered', // Can reference a field value (requires source), will throw error if it doesn't exist
```

Comment:
is source a variable or a special keyword? also replace fieldValue with real field from Firestore structure table